src/shared/components/rentCard/rentCard.tsx

```diff
@@ -0,0 +1,27 @@
+interface IRentCardPropsModel {
+	children?: string;
+	styleClass: string;
+	isColored?: boolean;
+	bottomColor?: string;
+	bottomContent?: string;
```

Comment:
```suggestion
	children: string;
	styleClass: string;
	isColored?: boolean;
	bottomColor?: string;
	bottomContent: string;
```
add Optional Chaining (?) where needed and avoid to use where not

Reply:
Done